cmd/rpcfuzz/rpcfuzz.go

```diff
@@ -1656,6 +1676,84 @@ func GetCurrentChainID(ctx context.Context, rpcClient *rpc.Client) (*big.Int, er
 	return chainId, err
 }
 
+func CallRPCAndValidate(ctx context.Context, rpcClient *rpc.Client, currTest RPCTest) testreporter.TestResult {
+	n := 1
+	currTestResult := testreporter.TestResult{
+		Name:             currTest.GetName(),
+		Method:           currTest.GetMethod(),
+		Args:             make([][]interface{}, n),
+		Result:           make([]interface{}, n),
+		Errors:           make([]error, n),
+		NumberOfTestsRan: n,
+	}
```

Comment:
I would say, if you're going to go through the trouble of creating a type of test results, you might want to have an actual constructor to avoid this constructions.

Reply:
added! 🧠 💡